src/MoBi.Presentation/Nodes/TreeNodeFactory.cs

```diff
@@ -107,6 +123,38 @@ public ITreeNode CreateFor(SimulationConfiguration simulationConfiguration)
          return buildConfigNode;
       }
 
+      public ITreeNode CreateFor(ModuleConfiguration moduleConfiguration)
+      {
+         var moduleConfigurationNode = new ModuleConfigurationNode(moduleConfiguration).WithIcon(ApplicationIcons.Module);
+         var module = moduleConfiguration.Module;
+
+         addModuleBuildingBlocks(moduleConfigurationNode, module);
+         
+         if(moduleConfiguration.SelectedMoleculeStartValues != null)
+            createAndAddNodeUnder(moduleConfigurationNode, moduleConfiguration.SelectedMoleculeStartValues);
+         if(moduleConfiguration.SelectedParameterStartValues != null)
+            createAndAddNodeUnder(moduleConfigurationNode, moduleConfiguration.SelectedParameterStartValues);
```

Comment:
I feel that resharper needs to be run in this file. Stuff seems to be a bit out of space alignment